scripts/examples/gn_efr32_example.sh

```diff
@@ -118,6 +118,10 @@ if [ "$#" == "0" ]; then
             Currently : v1.0-<branchName>-<ShortCommitSha>
         --release
             Remove all logs and debugs features (including the LCD). Yield the smallest image size possible
+        --slc_generate
+            Generate files with SLC for current board and options Requires an SLC-CLI installation or running in Docker.
+        --use_slc_files
```

Comment:
Nitpick: I'd be happier if this is called `--slc_reuse`, or at least keep any SLC-related arguments in the format --slc_<do_something>

Reply:
Will do